fusion/swift/.gitignore

```diff
@@ -0,0 +1,27 @@
+# Build generated
```

Comment:
This example deserves mention in the root `README.md`.

@tomelm or @d9chen , were one of you going to add that yourself or should @SSheldon do it in this pull request?